### DISCUSSION TITLE
Easily Bypassable with 2 clicks

### DISCUSSION THREAD

**Discussion by originaldreamer** [jordwest/news-feed-eradicator#406] [Ideas]
Whenever seeing news feed eradicator, a link appears. When you click on this link, it takes you very conveniently to a page where you can just disable the news feed eradicator.

step1:
![image](https://github.com/jordwest/news-feed-eradicator/assets/134401467/772bfaf4-08b0-4042-8e0d-96dbdb086ba3)

step2:
<img width="675" alt="image" src="https://github.com/jordwest/news-feed-eradicator/assets/134401467/3fcc8fa9-115e-4cb4-82a5-355963c39cf6">

step3:
![image](https://github.com/jordwest/news-feed-eradicator/assets/134401467/6d0121f9-7727-4755-85bc-15b0bb082b8a)

it is that easy to procrastinate with news feed eradicator. 

Please make so that this link can be hidden from the settings, it also makes it a lot cleaner without the link.

**Comment by drmercer** [comment]
Any browser extension can be disabled in a few clicks using the browser UI, so there's only so much an extension can do on this front.

Have you tried using the "Show feed for `x` mins" feature instead? That would at least turn off the feed again after a few minutes, which might help reduce procrastination
